lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
+        y : scalar or array-like
 
-        xerr/yerr : scalar or array-like, shape(n,1) or shape(2,n), optional
-            If a scalar number, len(N) array-like object, or an Nx1
+        xerr/yerr : scalar or array-like, shape(N,) or shape(2,N), optional
```

Comment:
Yes. I did some checks and the docs are 99% correct. I am submitting an issue right now about an inconsistency that I found.